internal/command/output.go

```diff
@@ -75,6 +77,25 @@ func (c *OutputCommand) Outputs(statePath string) (map[string]*states.OutputValu
 		return nil, diags
 	}
 
+	// See if we are using the Cloud backend
+	cloud, cloudOk := b.(*cloud.Cloud)
+	if cloudOk {
+		workspaceOutputs, err := cloud.WorkspaceOutputs(env)
+		if err != nil {
+			diags = diags.Append(fmt.Errorf("Failed to load state: %s", err))
```

Comment:
technically we failed to load the workspace

Reply:
should we return nil, diags here?  I think this will fail later if workspaceOutputs is nil

Reply:
Good catches